common/src/main/kotlin/com/hartwig/actin/molecular/evidence/actionability/IndirectEvidenceMatcher.kt

```diff
@@ -0,0 +1,107 @@
+package com.hartwig.actin.molecular.evidence.actionability
+
+import com.hartwig.actin.datamodel.molecular.driver.ProteinEffect
+import com.hartwig.actin.datamodel.molecular.driver.Variant
+import com.hartwig.actin.molecular.evidence.ServeVerifier.isCombinedProfile
+import com.hartwig.actin.molecular.evidence.curation.ApplicabilityFiltering
+import com.hartwig.actin.molecular.evidence.curation.GenericInhibitorFiltering
+import com.hartwig.actin.molecular.interpretation.GeneAlterationFactory
+import com.hartwig.serve.datamodel.Knowledgebase
+import com.hartwig.serve.datamodel.ServeRecord
+import com.hartwig.serve.datamodel.efficacy.EfficacyEvidence
+import com.hartwig.serve.datamodel.molecular.hotspot.KnownHotspot
+import com.hartwig.serve.datamodel.molecular.hotspot.VariantAnnotation
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+
+data class TreatmentKey(
```

Comment:
Would rename to `GeneEffectKey` or `HotspotEffectKey`, this class isn't really related to treatments?